docs/changes/4.1.0.rst

```diff
@@ -14,9 +14,21 @@ on GitHub.
 
 .. NOTE(vytas): No changes to the supported platforms (yet).
 
-
```

Comment:
A diff still remains in this file against `master`. Please revert all your changes to this release template.